build/generate-svg.js

```diff
@@ -60,7 +77,10 @@ async function main() {
     ...processFiles('node_modules/@primer/octicons/build/svg/*-16.svg', {prefix: 'octicon'}),
     ...processFiles('web_src/svg/*.svg'),
     ...processFiles('public/img/gitea.svg', {fullName: 'gitea-gitea'}),
+    ...processFiles('vscode-material-icon-theme/icons/*.svg', {prefix: 'material'}),
   ]);
+
+  await rm('vscode-material-icon-theme', {recursive: true, force: true});
```

Comment:
Seems a bit like overkill to me to clone a whole repo, copy all files, and then delete it again.
Is there no better way to achieve that?

Reply:
The key thing here is the `npm run build` that generates the JSON. It's not published so we have to generate it. I don't mind maintaining a git repo with a cron action that generates these files to some object storage if you prefer... Then here we will download a tar with the JSON + icons

Reply:
So stick with this or the other approach with the new repo?

Reply:
Would prefer a icon theme that is published somewhere, ideally npm 😉